noxfile.py

```diff
@@ -97,7 +97,7 @@ def lint(session):
         "nox",
     )
     files = ["nox", "tests", "noxfile.py", "setup.py"]
-    session.run("black", "--check", *files)
+    session.run("black", *files)
```

Comment:
I think this was committed by mistake?

```suggestion
    session.run("black", "--check", *files)
```

Reply:
How embarrassing...😅

Thanks for catching that!!